.eslintrc.js

```diff
@@ -8,125 +8,120 @@ module.exports = {
   },
   root: true,
   parser: '@typescript-eslint/parser',
-  extends: ['prettier', 'eslint:recommended'],
+  extends: ['plugin:@typescript-eslint/all', 'prettier', 'prettier/@typescript-eslint'],
   plugins: ['@typescript-eslint', 'prettier', 'import', 'unicorn'],
   rules: {
+    // unicorn rules
     'unicorn/filename-case': [
       'error',
       {
         'case': 'kebabCase'
       }
     ],
+
+    // import rules
     'import/newline-after-import': 1,
     'import/order': [
-      "error",
+      'error',
       {
-        "newlines-between": "always-and-inside-groups",
-        "groups": ["builtin", "external", "internal", "parent", "sibling", "index"]
+        'newlines-between': 'always-and-inside-groups',
+        'groups': ['builtin', 'external', 'internal', 'parent', 'sibling', 'index']
       }
     ],
     'import/no-default-export': 'error',
+
+    // prettier rules
     'prettier/prettier': ['error'],
```

Comment:
Is this still needed? Isn't it included by "prettier/@typescript-eslint"?

Reply:
yea, you might be right, ill check it :)

Reply:
I think it does not include. Removing it disables 'prettier help' so i guess it has to stay. prettier/@typescript-eslint only disables additional stuff